spec/plans/results.fmf

```diff
@@ -8,8 +8,12 @@ description: |
     and custom test results are required to follow it when creating their
     ``results.yaml`` file.
 
-    Results are saved as a YAML file, containing a single list of mappings,
-    one mapping describing a single test result.
+    Tests may choose JSON instead of YAML for their custom results file and
```

Comment:
what is this good for? Seems complicating thing, would not be simpler just to support yaml, or are there some requirements for this?

Reply:
Apparently, some users are fond of JSON: https://github.com/teemtee/tmt/pull/1961

Reply:
it would be better to link the original issue with the MR, for more context in that case

Reply:
I didn't feel it was necessary, as I'm not adding this text, I'm just moving it between files, slightly rephrasing it. `results.json` was already mentioned in the docs, therefore it didn't occur to me to point it out.